public/js/user.js

```diff
@@ -359,6 +360,62 @@ apos.define('apostrophe-workflow', {
       });
     };
 
+    self.enableForceExportRelated = function() {
+      apos.ui.link('apos-workflow-force-export-related', null, function($el, id) {
+        self.forceExportRelated(id);
+      });
+    };
+
+    self.forceExportRelated = function(id, callback) {
+      self.commitAllRelated = false;
+      self.skipAllRelated = false;
+      return apos.areas.saveAllIfNeeded(function() {
+        return self.getEditable({ ids: [ id ], related: true }, function(err, result) {
+          if (err) {
+            return callback(err);
+          }
+          var all = result.modified.concat(result.unmodified).filter(function(_id) {
+            return _id !== id;
+          });
+          if (!all.length) {
+            apos.notify('There were no related documents.', { dismiss: true });
+            return callback && callback(null);
+          }
+          return async.eachSeries(all, function(id, callback) {
```

Comment:
My only suggestion here would be to change the variable name of `id` here or on line 369, similarly to in `self.exportSimilarly`. This looks like it'll be fine, but sharing the variable seems harder to maintain if nothing else.